converse.js

```diff
@@ -4375,7 +4375,10 @@
                 $(iq).children('query').find('item').each(function (idx, item) {
                     this.updateContact(item);
                 }.bind(this));
-                return true;
+
+                converse.emit('rosterPush', iq);
+
```

Comment:
Please document this emitted event, here: https://github.com/jcbrand/converse.js/blob/master/docs/source/development.rst#event-types